package.json

```diff
@@ -78,7 +78,7 @@
     "postcss-loader": "1.2.1",
     "prompt": "1.0.0",
     "react-addons-test-utils": "15.4.1",
-    "react-hot-loader": "^1.3.1",
+    "react-hot-loader": "3.0.0-beta.6",
```

Comment:
@nickytonline To answer the question, Yeah RR4 works with react-hot-loader 1.3.1.
@oshalygin That makes sense.  I was wondering...
@coryhouse Happy to contribute!! :)